rllib/tuned_examples/ppo/unity3d-soccer-strikers-vs-goalie-ppo.yaml

```diff
@@ -0,0 +1,46 @@
+# NOTE: This example will not run w/o a proper config.multiagent setup,
```

Comment:
Can also link to this from the ppo algorithm section in the docs.

Reply:
Sure. +1